data/sql/updates/pending_db_world/rev_1617974280334682621.sql

```diff
@@ -0,0 +1,25 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1617974280334682621');
+
+-- Splintered Skeleton
+SET @ENTRY := 10478;
+
+-- Set template ai to SAI
+DELETE FROM `creature_template` WHERE (`entry` = @ENTRY);
+INSERT INTO `creature_template` (`entry`, `difficulty_entry_1`, `difficulty_entry_2`, `difficulty_entry_3`, `KillCredit1`, `KillCredit2`, `modelid1`, `modelid2`, `modelid3`, `modelid4`, `name`, `subname`, `IconName`, `gossip_menu_id`, `minlevel`, `maxlevel`, `exp`, `faction`, `npcflag`, `speed_walk`, `speed_run`, `scale`, `rank`, `dmgschool`, `DamageModifier`, `BaseAttackTime`, `RangeAttackTime`, `BaseVariance`, `RangeVariance`, `unit_class`, `unit_flags`, `unit_flags2`, `dynamicflags`, `family`, `trainer_type`, `trainer_spell`, `trainer_class`, `trainer_race`, `type`, `type_flags`, `lootid`, `pickpocketloot`, `skinloot`, `PetSpellDataId`, `VehicleId`, `mingold`, `maxgold`, `AIName`, `MovementType`, `InhabitType`, `HoverHeight`, `HealthModifier`, `ManaModifier`, `ArmorModifier`, `RacialLeader`, `movementId`, `RegenHealth`, `mechanic_immune_mask`, `spell_school_immune_mask`, `flags_extra`, `ScriptName`, `VerifiedBuild`) VALUES
+(@ENTRY, 0, 0, 0, 0, 0, 11401, 9788, 9789, 9790, 'Splintered Skeleton', NULL, NULL, 0, 59, 60, 0, 233, 0, 1, 1.71429, 1, 1, 0, 3.5, 1000, 2000, 1, 1, 1, 64, 2048, 0, 0, 0, 0, 0, 0, 6, 0, 10478, 10478, 0, 0, 0, 488, 643, 'SmartAI', 1, 3, 1, 3, 3, 1, 0, 164, 1, 8602131, 0, 0, '', 12340);
```

Comment:
make an update instead of delete & insert for creature_template, unless you are adding a completely new creature

Reply:
Removed this query since it was [duplicate](https://github.com/azerothcore/azerothcore-wotlk/pull/5244/files/7d5188033684a13b2c5e3d0e39ba7150963902fb#diff-3d7c58eafb49c26055d4e311ee92017d1952009be6003ab17bdc5065986a25ddR12)